faasmtools/env.py

```diff
@@ -3,6 +3,8 @@
 
 PROJ_ROOT = dirname(dirname(abspath(__file__)))
 THIRD_PARTY_DIR = join(PROJ_ROOT, "third-party")
+WASM_DIR = "/usr/local/faasm/wasm"
+FAASM_RUNTIME_ROOT = "/usr/local/faasm/runtime_root"
```

Comment:
This dir is now hard-coded in quite a few places, but it's preferable to having Python dependencies across all the modules just for these constants.

Reply:
Indeed, this was making `inv -l` fail in so many places unnecessarily.